cmd/kubectl-gadget/collector.go

```diff
@@ -42,6 +43,7 @@ var socketCollectorCmd = &cobra.Command{
 
 var (
 	processCollectorParamThreads bool
+	socketCollectorProto         string
```

Comment:
This can seem odd but it took me a bit of time to realize "Proto" is about "Protocol" (TCP/UDP).
Do you mind running something like this: `s/((P|p)roto)/$1col`?

Reply:
I updated it to be `protocol` everywhere but let the CLI option to be `proto` as it's shorter. Any thoughts @blanquicet ?

Reply:
I think it is ok, some Linux networking tools also use `proto` because it is more user-friendly when using the command line. In any case, if users are not sure what it is, the `--help` will describe it properly.